PayKids/presentation/src/main/java/com/paykids/presentation/view/home/HomeActivity.kt

```
@@ -0,0 +1,16 @@
+package com.paykids.presentation.view.home
+
+import com.paykids.presentation.base.BaseActivity
+import com.paykids.presentation.databinding.ActivityHomeBinding
+import dagger.hilt.android.AndroidEntryPoint
+
+@AndroidEntryPoint
+class HomeActivity : BaseActivity<ActivityHomeBinding>() {
+
+    override fun initView() {
+    }
```

detekt (1.23.7)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)